modules/Bio/EnsEMBL/Analysis/Hive/Config/LoadAssembly.pm

```diff
@@ -0,0 +1,557 @@
+    refseq_import_ftp_path  => $self->o('refseq_base_ftp').'/#assembly_refseq_accession#_#assembly_name#_genomic.gff.gz',
+    refseq_mrna_ftp_path    => $self->o('refseq_base_ftp').'/#assembly_refseq_accession#_#assembly_name#_rna.fna.gz',
+    refseq_report_ftp_path => $self->o('refseq_base_ftp').'/#assembly_refseq_accession#_#assembly_name#_assembly_report.txt',
+
+
+########################
+# db info
+########################
+    reference_db => {
+      -dbname => $self->o('reference_db_name'),
+      -host   => $self->o('reference_db_server'),
+      -port   => $self->o('reference_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    production_db => {
+      -host   => $self->o('production_db_server'),
+      -port   => $self->o('production_db_port'),
+      -user   => $self->o('user_r'),
+      -pass   => $self->o('password_r'),
+      -dbname => 'ensembl_production',
+      -driver => $self->o('hive_driver'),
+    },
+
+    taxonomy_db => {
+      -host   => $self->o('production_db_server'),
+      -port   => $self->o('production_db_port'),
+      -user   => $self->o('user_r'),
+      -pass   => $self->o('password_r'),
+      -dbname => 'ncbi_taxonomy',
+      -driver => $self->o('hive_driver'),
+    },
+  };
+}
+
+sub pipeline_create_commands {
+  my ($self) = @_;
+
+  return [
+  # inheriting database and hive tables' creation
+    @{$self->SUPER::pipeline_create_commands},
+    'mkdir -p '.$self->o('genome_dumps'),
+  ];
+}
+
+
+sub pipeline_wide_parameters {
+  my ($self) = @_;
+
+  return {
+    %{$self->SUPER::pipeline_wide_parameters},
+    load_toplevel_only => $self->o('load_toplevel_only'),
+  }
+}
+
+
+## See diagram for pipeline structure
+sub pipeline_analyses {
+  my ($self) = @_;
+
+  return [
+    {
+      -logic_name => 'create_core_db',
+      -module     => 'Bio::EnsEMBL::Analysis::Hive::RunnableDB::HiveCreateDatabase',
+      -parameters => {
+        'target_db'        => $self->o('reference_db'),
+        'enscode_root_dir' => $self->o('enscode_root_dir'),
+        'create_type'      => 'core_only',
+      },
+      -rc_name    => 'default',
+      -flow_into  => {
+        1 => ['populate_production_tables'],
+      },
+    },
+
+    {
+      -logic_name => 'populate_production_tables',
+      -module     => 'Bio::EnsEMBL::Analysis::Hive::RunnableDB::HiveAssemblyLoading::HivePopulateProductionTables',
+      -parameters => {
+        'target_db'        => $self->o('reference_db'),
+        'output_path'      => $self->o('output_path'),
+        'enscode_root_dir' => $self->o('enscode_root_dir'),
+        'production_db'    => $self->o('production_db'),
+      },
+      -rc_name    => 'default',
+      -flow_into  => {
+        1 => WHEN ('#load_toplevel_only# == 1' => ['process_assembly_info'],
+            '#load_toplevel_only# == 2' => ['custom_load_toplevel'],
```

Comment:
```suggestion
            '#load_toplevel_only# == 2' => ['custom_load_toplevel'],
```
It's fine, maybe a perltidy indentation issue? but not sure if easy to configure.

Reply:
I didn't use perltidy, I added indentation but maybe not how you were expected. I see the suggestion as being the same code as before